dev/docker/build-docker.sh

```diff
@@ -122,14 +134,18 @@ fi
 cd ${script_dir}/${component_type}
 if [[ "${platform_type}" == "all" ]]; then
   if [ ${build_latest} -eq 1 ]; then
-    docker buildx build --no-cache --pull --platform=linux/amd64,linux/arm64 ${build_args} --push --progress plain -f Dockerfile -t ${image_name}:latest -t ${image_name}:${tag_name} .
+    # FIXME: need to add back --no-cache
```

Comment:
@unknowntpo Do we need to roll back the changed `--no-cache` in this PR?

Reply:
rollbacked.